### PR TITLE
Browser animation adjustments, code cleanup

### PR DESCRIPTION
**Pull request by christianbaroni**
## What changed (plus any additional context for devs)
- Smooths out WebView height resizing
  - The WebView now grows to fill its container instead of having a fixed height, which allows indirectly animating the WebView height by animating the container height
- Ensures tab close gestures are always reset — previously there was a rare bug where for certain gesture sequences, the gesture wouldn't be reset
- Patches `faster-image` to add a `topResize` resize mode on iOS. This allows tab screenshots to be pinned to the top of the WebView container and display correctly regardless of the height of the screenshot (screenshots are taller if taken when the WebView is expanded). The WebView only expands on iOS so this isn't needed on Android.
- Updates the `cachePolicy` for browser screenshots from `'discNoCacheControl'` to `'memory'` as the images are already stored locally.
- Removes the `description` field from the dapps object used for browser search as it's not used — this cuts down the size from 2.2 MB to 0.9 MB
- Removes unused code in `createRainbowStore`
- Removes commented code

### CODE REVIEW

File: src/resources/metadata/dapps.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function useDapps(config?: UseQueryOptions<Dapp[]>): { dapps: Dapp[] } {
           .map(dapp => ({
             name: dapp!.name,
             shortName: dapp!.shortName,
-            description: dapp!.description,
+            description: '', // Remove to cut down on total object size
```
Comment (walmat) [inline src/resources/metadata/dapps.tsx:48]: can we just remove the key itself if it's not going to be used?

**Review by walmat — COMMENTED** [review]
code looks good, will run on both OSes to make sure it's solid tonight

**Review by walmat — APPROVED** [review]
LGTM. nice work man

**Review by derHowie — APPROVED** [review]

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/@christian/browser-animation-adjustments/334243d74a115eeba24c11da35f0e89822fc4008.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/@christian/browser-animation-adjustments/334243d74a115eeba24c11da35f0e89822fc4008.ipa) for 334243d74a115eeba24c11da35f0e89822fc4008

**Pull request merged**